docs/source/apps/projinfo.rst

```diff
@@ -201,6 +192,24 @@ The following control parameters can appear in any order:
 
     .. note:: only used for coordinate operation computation
 
+.. option:: --show-superseded
+
+    When enabled, coordinate operations that are superseded by others will be
+    listed. Note that supersession is not equivalent to deprecation: superseded
+    operations are still considered valid although they have a better equivalent,
+    whereas deprecated operations have been determined to be erroneous and are
+    not considered at all.
+
+    .. note:: only used for coordinate operation computation
+
+.. option:: --hide-ballpark
+
+    .. versionadded:: 7.1
+
+    Hides any coordinate operation that is, or contains, a ballpark transformation.
```

Comment:
It would nice if "ballpark" contained a link to a description of the "ballpark transformation" concept. I am not sure if have a proper description on the website. If not it could be put in the glossary and linked here with ```:term:`ballpark` ```

Reply:
Done in commits ef3a387abd5989f6d5242a6c8cd0f7b6e7318484 and 335d9367fd07ebe5da50d2f9e336d5d8f6e31aee . I'll cherry-pick ef3a387abd5989f6d5242a6c8cd0f7b6e7318484 to 7.0 branch